login/routers/middlewares/SignInValidator.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+
+const router = express.Router();
+const LoginRequest = require('../../db/db_objects/LoginRequest.js');
+const signInValidation = require('../../core/validations/SignInValidation.js');
+
+
+router.post('/', (request, response,next) => {
+    const loginRequest = new LoginRequest(request.body.login, request.body.password);
+    if(signInValidation.SignInValidation(loginRequest, response)){
```

Comment:
Maybe rename the SignInValidation method to validate? Would transfer the intent clearer.